.github/workflows/pytest-workflow_release.yml

```diff
@@ -0,0 +1,171 @@
+name: pytest-workflow
```

Comment:
Doesn't the "system" (GitHub?) require the workflows to have unique names?  (Perhaps not when they apply to different branches 🤔 )